### PR TITLE
Feature/namespacing and log cleaning

### PR DESCRIPTION
**Pull request by pmbauer**
I'm not done with this by any means, but want to get it out there to get some feedback.

### CODE REVIEW

**Comment by pmbauer** [comment]
individual diffs:
https://github.com/udacity/logspout-logstash/pull/2/commits/76901433687449d2ab3634c680f9fec7e611f00d

https://github.com/udacity/logspout-logstash/pull/2/commits/0778b0e25e4923883de0dda5e14d9c76da9eff9a

File: logstash.go [hunk 1]
```diff
@@ -231,48 +239,82 @@ func (a *LogstashAdapter) sendMessage(msg *router.Message) error {
...
-func serialize(msg *router.Message) ([]byte, error) {
-	var js []byte
-	var jsonMsg map[string]interface{}
-
-	dockerInfo := DockerInfo{
-		Name:     msg.Container.Name,
-		ID:       msg.Container.ID,
-		Image:    msg.Container.Config.Image,
-		Hostname: msg.Container.Config.Hostname,
+func (a *LogstashAdapter) serialize(msg *router.Message) ([]byte, error) {
```
Comment (jamesbibby) [inline logstash.go]: Nice conversion to a struct method

File: logstash.go [hunk 2]
```diff
@@ -231,48 +239,82 @@ func (a *LogstashAdapter) sendMessage(msg *router.Message) error {
...
+	var collapsed string
+	collapsedBuffer, err := json.Marshal(node)
+	if err != nil {
+		collapsed = fmt.Sprintf("%v", node)
+	} else {
+		collapsed = string(collapsedBuffer)
+	}
+	return truncateField(collapsed)
+}
+
```
Comment (jamesbibby) [inline logstash.go]: Was this supposed to do something other than just return itself?

Comment (pmbauer) [inline logstash.go]: Yes :) and much thanks for the review

**Pull request merged**